### PR TITLE
update dex operator for 0.2.1

### PR DESCRIPTION
**Pull request by chrisahl**
Signed-off-by: Chris Ahl <cahl@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,12 +376,12 @@ date -u "+0.0.0-%Y%m%d-%H-%M-%S"
 1. In the **Find or create a new tag field**, paste the value from the date command above
 1. Select **Create new tab on publish**
 1. Select **Publish release**.
-   This will cause a github action to start.  Once the github action is complete, move on to
-   the next part which will pull the new dex-operator quay image from https://quay.io/repository/identitatem/dex-operator?tab=tags
+   This will cause a github action to start.  Once the github action is complete, the new dex-operator quay image from https://quay.io/repository/identitatem/dex-operator?tab=tags.  Once the dex-operator quay image is available, move on to
```
Comment (robinbobbitt) [inline README.md:379]: @chrisahl please take a quick peek at this sentence and tweak it to read correctly. Thanks!

File: bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ metadata:
       for Kubernetes", "multicluster engine for Kubernetes"]'
     operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
-  name: idp-mgmt-operator.v0.0.1
+  name: idp-mgmt-operator.v0.0.0-20220115-00-41-28
```
Comment (robinbobbitt) [inline bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml:16]: Please revert.

File: bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -535,5 +535,4 @@ spec:
   provider:
     name: Red Hat
     url: https://github.com/identitatem
-  replaces: idp-mgmt-operator.v0.2.0
```
Comment (robinbobbitt) [inline bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml:538]: Please revert these changes.

**Review by robinbobbitt — CHANGES_REQUESTED** [review]
Looks good but please see comments and shout if we need to discuss. Thanks!

File: bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (chrisahl) [reply bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml:538]: Current `make bundle` did this.   I added more to README to help remind us to revert.

File: bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (chrisahl) [reply bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml]: Current `make bundle` did this.  I added more to README to help remind us to revert

**Review by robinbobbitt — APPROVED** [review]
Thanks @chrisahl ! lgtm

**Review by robinbobbitt — APPROVED** [review]
